frontend/src/store/facilityBooking/action.ts

```diff
@@ -128,13 +137,33 @@ export const getMyBookings = (userId: string) => async (dispatch: Dispatch<Actio
   dispatch(setIsLoading(false))
 }
 
+/**
+ *
+ * Takes in a number and closes the booking if it is -1
+ *
+ * @params isDeleteMyBooking (number, optional)
+ * @returns updates `isDeleteMyBooking`
+ *
+ * @remarks
+ * <any remarks on this function put here>
+ */
 // -1 stands for closed, any others means open for that specific ID.
```

Comment:
do consolidate all these stray remarks under `@remarks`. Also can help me add the TODO comment in also thanks
```suggestion
 * @remarks
 * -1 stands for closed, any others means open for that specific ID.
 * // TODO shouldn't `isDeleteMyBooking` be a boolean value ??
 */

```